zerver/tests/test_create_video_call.py

```diff
@@ -187,24 +187,24 @@ def test_create_bigbluebutton_link(self) -> None:
     def test_join_bigbluebutton_redirect(self) -> None:
         responses.add(
             responses.GET,
-            "https://bbb.example.com/bigbluebutton/api/create?meetingID=zulip-1&name=general%2520%253E%2520meeting&moderatorPW=a&attendeePW=76af7efae0d034d1e3335ed1b90f24b6cadf2bf1&checksum=check",
-            "<response><returncode>SUCCESS</returncode><messageKey/></response>",
+            "https://bbb.example.com/bigbluebutton/api/create?meetingID=zulip-1&name=general%20%3E%20meeting&moderatorPW=a&attendeePW=76af7efae0d034d1e3335ed1b90f24b6cadf2bf1&checksum=check",
+            "<response><returncode>SUCCESS</returncode><messageKey/><createTime>0</createTime></response>",
         )
         response = self.client_get(
             "/calls/bigbluebutton/join",
             {
                 "meeting_id": "zulip-1",
                 "password": "a",
                 "checksum": "check",
-                "name": "general%20%3E%20meeting",
+                "name": "general > meeting",
```

Comment:
Does this change belong in the previous commit?

Reply:
I did ran the tests for each commit.
But I agree that it seems like this should be in the commit before.
I will test this tomorrow.